src/combinatorics/quiver/basic.lean

```diff
@@ -86,17 +86,29 @@ def comp {U : Type*} [quiver U] {V : Type*} [quiver V] {W : Type*} [quiver W]
 { obj := λ X, G.obj (F.obj X),
   map := λ X Y f, G.map (F.map f), }
 
+@[simp] lemma comp_id {U : Type*} [quiver U] {V : Type*} [quiver V] (F : prefunctor U V) :
+  F.comp (id _) = F := by { cases F, refl, }
+
+@[simp] lemma id_comp {U : Type*} [quiver U] {V : Type*} [quiver V] (F : prefunctor U V) :
+  (id _).comp F = F := by { cases F, refl, }
+
 @[simp]
 lemma comp_assoc
   {U V W Z : Type*} [quiver U] [quiver V] [quiver W] [quiver Z]
   (F : prefunctor U V) (G : prefunctor V W) (H : prefunctor W Z) :
   (F.comp G).comp H = F.comp (G.comp H) :=
 begin
-  apply prefunctor.ext, rotate,
-  { rintro X, refl, },
-  { rintro X Y Z, refl, }
+  fapply prefunctor.ext,
+  exact (λ X, rfl),
+  exact (λ X Y Z, rfl),
 end
 
+infix ` ⟶q `:50 := prefunctor
+
+infix ` ≫q `:50 := prefunctor.comp
```

Comment:
```suggestion
infix ` ⋙q `:50 := prefunctor.comp
```
for consistency with `functor.comp`.

I'm a little skeptical of the need for notation here, but perhaps you've got bigger plans for quivers in mind?

Reply:
Well, we plan on adding coverings of quivers and using quivers for cayley graphs, and at some point it just becomes frustrating not to have notation for prefunctors. Should `prefunctor` be written as `\functor`+`q` then?

Reply:
No, that's okay. I wasn't clear before on the scope of your plans, and wanted to make sure we weren't just adding notation to a barely used file!